doc/src/build/package-upgrades.md

```diff
@@ -0,0 +1,283 @@
+  TransferObjects([Result(0)],Input(0)),
+]
+
+Sender: <PUBLISHER-ID>
+Gas Payment: Object ID:, version: 0x6, digest: HLAcq3SFPZm4xvcPryXk5MjA718xGVnTYCdtWbFsaJpe 
+Gas Owner: <PUBLISHER-ID>
+Gas Price: 1
+Gas Budget: <GAS-BUDGET-AMOUNT>
+
+----- Transaction Effects ----
+Status : Success
+Created Objects:
+  - ID: <ORIGINAL-PACKAGE-ID> , Owner: Immutable
+  - ID: <UPGRADE-CAP-ID> , Owner: Account Address ( <PUBLISHER-ID> )
+  - ID: <PUBLISHER-ID> , Owner: Account Address ( <PUBLISHER-ID> )
+Mutated Objects:
+  - ID: <GAS-COIN-ID> , Owner: Account Address ( <PUBLISHER-ID> )
+
+----- Events ----
+Array []
+----- Object changes ----
+Array [
+    Object {
+        "type": String("mutated"),
+        "sender": String("<PUBLISHER-ID>"),
+        "owner": Object {
+            "AddressOwner": String("<PUBLISHER-ID>"),
+        },
+        "objectType": String("0x2::coin::Coin<0x2::sui::SUI>"),
+        "objectId": String("<GAS-COIN-ID>"),
+        "version": Number(7),
+        "previousVersion": Number(6),
+        "digest": String("6R39f68p4tGqJWJTakKCyL8tz2w2XTvJ3Mu5nGwxadda"),
+    },
+    Object {
+        "type": String("published"),
+        "packageId": String("<ORIGINAL-PACKAGE-ID>"),
+        "version": Number(1),
+        "digest": String("FrBhLF2Rn4jP3SUsss7aXqwDDRtoKxgGbPm8eVkH7jrQ"),
+        "modules": Array [
+            String("sui_package"),
+        ],
+    },
+    Object {
+        "type": String("created"),
+        "sender": String("<PUBLISHER-ID>"),
+        "owner": Object {
+            "AddressOwner": String("<PUBLISHER-ID>"),
+        },
+        "objectType": String("0x2::package::UpgradeCap"),
+        "objectId": String("<UPGRADE-CAP-ID>"),
+        "version": Number(7),
+        "digest": String("BoGQ63r27DFZDMC8p7YwRcDpToFpbZ9rG1R4o4uXkaUw"),
+    },
+    Object {
+        "type": String("created"),
+        "sender": String("<PUBLISHER-ID>"),
+        "owner": Object {
+            "AddressOwner": String("<PUBLISHER-ID>"),
+        },
+        "objectType": String("<ORIGINAL-PACKAGE-ID>::sui_package::<MODULE-NAME>"),
+        "objectId": String("<PUBLISHER-ID>"),
+        "version": Number(7),
+        "digest": String("BC3KeuATKJozLNipbUz2GWzoDXbodXH4HLm59TxJSmVd"),
+    },
+]
+----- Balance changes ----
+Array [
+    Object {
+        "owner": Object {
+            "AddressOwner": String("<PUBLISHER-ID>"),
+        },
+        "coinType": String("0x2::sui::SUI"),
+        "amount": String("-9328480"),
+    },
+]
+```
+
+The result includes an **Object changes** section with two pieces of information you need for upgrading, an `UpgradeCap` ID and your package ID. 
+
+You can identify the different objects using the `Object.objectType` value in the response. The `UpgradeCap` entry has a value of `String("0x2::package::UpgradeCap")` and the `objectType` for the package reads `String("<PACKAGE-ID>::sui_package::<MODULE-NAME>")`
```

Comment:
it calls out package ID but then says `objectType` - it took me a minute to see PACKAGE ID in the string. maybe call out that the  objectType includes the package ID?